### DISCUSSION TITLE
My commits are being recognised as 2 different users

### DISCUSSION THREAD

**Discussion by jakemarsdenn** [community/community#135116] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Whenever I make a commit, its recognised as "Jake Marsden and Jake Marsden <commit-name>". Has anyone has this issue before?

<img width="910" alt="Screenshot 2024-08-08 at 10 30 37 PM" src="https://github.com/user-attachments/assets/fece6477-837b-4f75-9ba0-bbc10a3d667e">

**Comment by jvcByte** [comment, marked as answer]
Yes, this issue can occur when your Git configuration has multiple entries for the user name or email, causing Git to recognize two different identities for the same person. Here's how you can resolve it:

1. **Check Your Global Git Configuration:**
   Open a terminal and run:
   ```bash
   git config --global --list
   ```
   Look for multiple `user.name` or `user.email` entries. If you find duplicates or extra spaces in the name or email, that's likely the cause.

2. **Fix the Configuration:**
   To fix it, you can set your name and email correctly:
   ```bash
   git config --global user.name "Jake Marsden"
   git config --global user.email "your.email@example.com"
   ```
   This will overwrite any previous entries with the correct information.

3. **Check the Local Repository Configuration:**
   If the issue persists, check the local repository's configuration:
   ```bash
   git config --local --list
   ```
   If there are conflicting entries here, you can remove or correct them using:
   ```bash
   git config --local --unset user.name
   git config --local --unset user.email
   ```

4. **Check for Multiple Authors in Commit Messages:**
   If you see both names in your commit history, it might be due to previous commits made with different configurations. You can amend the commits to fix this:
   ```bash
   git commit --amend --author="Jake Marsden <your.email@example.com>"
   ```

This should resolve the issue, ensuring that your commits are recognized with a single author name.

**Reply by awladux** [reply]
What is this

**Reply by jakemarsdenn** [reply]
This worked thank you!

**Reply by jvcByte** [reply]
It's a pleasure

**Comment by awladux** [comment]
My name is
